R/appMetadata.R

```diff
@@ -54,8 +54,10 @@ appMetadata <- function(appDir,
     appMode = appMode,
     appPrimaryDoc = appPrimaryDoc,
     hasParameters = hasParameters,
+    contentCategory = contentCategory,
     documentsHavePython = documentsHavePython,
-    quartoInfo = quartoInfo
+    quartoInfo = quartoInfo,
+    isCloudServer = isCloudServer
```

Comment:
`isCloudServer` feels out-of-place. It is a characteristic of the server, not the on-disk files. It's passed to `inferAppMode` so we can downgrade `rmd-static` to `rmd-shiny`.

Let's keep it as an argument to `createAppManifest`.